### PR TITLE
 replace deprecated function call

### PR DESCRIPTION
**Pull request by bumburoom**
replace deprecated function call

### CODE REVIEW

**Comment by markstory** [comment]
Looks like the minimum version for CakePHP will need to be raised for these changes to pass their tests.

**Comment by bumburoom** [comment]
Can I do this or does it have to be done by one of you guys?

File: src/QueryBuilder.php [hunk 1]
```diff
@@ -439,6 +439,25 @@ public function simpleQueryString($fields, $string)
...
+     * {{{
+     *  $builder->match('user.name', 'jose');
+     * }}}
+     *
+     * @param string $field The field to query by.
+     * @param string $value The match to find in field.
+     * @return \Elastica\Query\Match
+     * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html
+     */
+    public function match($field, $value)
```
Comment (saeideng) [inline src/QueryBuilder.php]: looks already exists in the `master`

**Comment by markstory** [comment]
@bumburoom you can do it. The composer.json file is part of this repository.

**Pull request closed without merging**